packages/elixir-ls/package.yaml

```
@@ -13,7 +13,7 @@ categories:
   - DAP
 
 source:
-  id: pkg:github/elixir-lsp/elixir-ls@v0.14.6
+  id: pkg:github/elixir-lsp/elixir-ls@v0.15.0
```

```suggestion
  id: pkg:github/elixir-lsp/elixir-ls-v0.15.0@v0.15.0
```

I don't know exactly how to format the Mason ids for packages, but the above suggestion seems sane?

---

after reading the purl spec, I think that it is "wrong" but that would mean to get this to work, the elixir-ls team needs to add a elixir-ls.zip to their release.